sync/metrics.go

```diff
@@ -186,6 +198,17 @@ func (m *metrics) newSubjectiveHead(ctx context.Context, height uint64, timestam
 	})
 }
 
+func (m *metrics) failedValidationAgainstSubjHead(ctx context.Context, height int64, hash string) {
```

Comment:
We call this case verification as opposed to validation, so lets keep it consistent.

Reply:
Done (see comment below).